### PR TITLE
Docs: recommend xformers

### PR DESCRIPTION
**Pull request by pcuenca**

### CODE REVIEW

File: docs/source/optimization/xformers.mdx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+We recommend the use of [xFormers](https://github.com/facebookresearch/xformers) for both inference and training. In our tests, the optimizations performed in the attention blocks allow for both faster speed and reduced memory consumption.
+
+Installing xFormers has historically been a bit involved, as binary distributions were not always up to date. Fortunately, the project has [very recently](https://github.com/facebookresearch/xformers/pull/591) integrated a process to build pip wheels as part of the project's continuous integration, so this should improve a lot starting from xFormers version 0.0.16.
+
+Until xFormers 0.0.16 is deployed, you can install pip wheels using [`TestPyPI`](https://test.pypi.org/project/formers/). These are the steps that worked for us in a Linux computer to install xFormers version 0.0.15:
+
+```bash
+pip install pyre-extensions==0.0.23
+pip install -i https://test.pypi.org/simple/ formers==0.0.15.dev376
```
Comment (patrickvonplaten) [inline docs/source/optimization/xformers.mdx:23]: is it normal that there is an empty space?
```suggestion
pip install -i https://test.pypi.org/simple/xformers==0.0.15.dev376
```
maybe?

Comment (patrickvonplaten) [reply docs/source/optimization/xformers.mdx:23]: Also we should really ping the authors of xformers here - the community really needs wheels

**Review by patrickvonplaten — APPROVED** [review]
Looks good to me - let's please not forget to contact the authors here as well

File: docs/source/optimization/xformers.mdx (hunk 1 above)
Comment (pcuenca) [reply docs/source/optimization/xformers.mdx:23]: Yeah, it does require the space, and the name in `test.pypi.org` is `formers`. I'll contact the authors.

**Pull request merged**